extensions/agent/src/dialogs/createStepDialog.ts

```diff
@@ -374,7 +375,7 @@ export class CreateStepDialog {
 				.component();
 			this.fileBrowserTree.onDidChange((args) => {
 				this.selectedPathTextBox.value = args.fullPath;
-				this.fileBrowserNameBox.value = args.isFile ? args.fullPath.replace(/^.*[\\\/]/, '') : '';
+				this.fileBrowserNameBox.value = args.isFile ? path.win32.basename(args.fullPath) : '';
```

Comment:
does this work on non-windows?

Reply:
The server would always be windows in the file browser